object/site_cache.go

```diff
@@ -105,6 +106,9 @@ func refreshSiteMap() error {
 		}
 
 		newSiteMap[strings.ToLower(site.Domain)] = site
+		if site.EnableAlert {
+			healthCheckNeededDomains = append(healthCheckNeededDomains, site.Domain)
```

Comment:
How to delete an item in healthCheckNeededDomains?

Reply:
Call refreshSiteMap function